examples/problem_decomposition/ifci_mifno.ipynb

```diff
@@ -0,0 +1,554 @@
+    "vqe = VQESolver({\n",
+    "    \"qubit_hamiltonian\": qu_op,\n",
+    "    \"ansatz\": hea_circ,\n",
+    "    \"initial_var_params\": \"zeros\",\n",
+    "    \"verbose\": True\n",
+    "})\n",
+    "vqe.build()\n",
+    "vqe.simulate()"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "### 5. Recomputing the total energy of the system\n",
+    "\n",
+    "In order to assess the impact of quantum workflows applied to one or several fragments on the total energy of the system, the helper class provides a `mi_summation` method. This class recomputes the total energy of the system, using the reference values obtained classically by QEMIST Cloud, or instead using the values obtained by your quantum workflows for the fragments you specify. This summation was described by the formulas in the first section of this notebook.\n",
+    "\n",
+    "If no argument is passed to `mi_summation`, then only reference values are used in the summation, which should then be in agreement (up to machine precision) with `fno_fragments.e_tot`, originally read from the QEMIST Cloud result blob. Otherwise, passing a dictionary using fragment labels as keys and the energy as the corresponding values does the trick. For MI-FNO fragments, the MP2 correction is automatically added, and a new total energy is computed. The difference between this new total energy and the reference total energy gives you a measure of the accuracy and the impact of combining quantum workflows with classical calculations performed with MI."
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": 9,
+   "metadata": {},
+   "outputs": [
+    {
+     "name": "stdout",
+     "output_type": "stream",
+     "text": [
+      "Reconstructed energy: -15.848867932624449\n",
+      "MIFNO QEMIST Cloud energy: -15.83647358459995\n",
+      "Difference: 0.01239434802449857\n"
+     ]
+    }
+   ],
+   "source": [
+    "e = fno_fragments.mi_summation({\"(1,)\": vqe.optimal_energy})\n",
+    "print(f\"Reconstructed energy: {e}\\nMIFNO QEMIST Cloud energy: {fno_fragments.e_tot}\\nDifference: {abs(e-fno_fragments.e_tot)}\")"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "## Closing words\n",
+    "\n",
+    "This feature allows us to bring together the state-of-the-art MI problem decomposition schemes available at scale in QEMIST cloud, and quantum workflows written with Tangelo. It facilitates exploring larger, more industrially-relevant use cases with a combination of classical and quantum workflows of your choice, on subproblems that are more ammenable to current devices.\n",
+    "\n",
+    "The method of increments combines the iFCI and MI-FNO workflows. While these two methods share similarities, they differ in certain aspects, such as the truncation of the virtual orbital space and the inclusion of a correction factor for MI-FNO fragments (which is not required for iFCI fragments)\n",
```

Comment:
See comment and reformulation from earlier

MI does not "combine" those two approaches, in english i think that suggest MI uses both at once, together

```
Both iFCI and MI-FNO implement the method of increments. They differ in how they truncate the virtual orbital space (SNO vs FNO) and the inclusion of a correction factor for MI-FNO fragments (which is not required for iFCI fragments)\n"
```